### DISCUSSION TITLE
Bright cities

### DISCUSSION THREAD

**Discussion by Menestt22** [TownyAdvanced/Towny#7408] [Q&A]
how to make cities on the towny-dynmap plugin look brighter and more contrasting
example : https://map.victorixy.one/#world;flat;7616,64,-5136;0

**Comment by Menestt22** [comment]
my map : http://d14.gamely.pro:20100/#world;flat;968,64,-7200;0

**Comment by LlmDl** [comment]
I don't know about this stuff, sorry.
